src/main/java/org/commcare/suite/model/QueryPrompt.java

```diff
@@ -181,13 +184,18 @@ public boolean isSelect() {
      * Evalualtes validation message against given eval context
      *
      * @param ec eval context to evaluate the validation message
-     * @return evaluated validation message or empty string if no validation message defined
+     * @return evaluated validation message or a default text if no validation message is defined
      */
     public String getValidationMessage(EvaluationContext ec) {
         if (validation != null && validation.getMessage() != null) {
             return validation.getMessage().evaluate(ec);
         }
-        return "";
+
+        try {
+            return Localization.get("invalid_answer_error");
```

Comment:
nit: more consistent naming:
```suggestion
            return Localization.get("case.search.answer.invalid");
```